### PR TITLE
Add K8s Job dedicated for formatting master journal

### PR DESCRIPTION
**Pull request by jiacheliu3**
This changes adds a set of Helm YAML templates.

The YAML creates a K8s Job that formats the master log and dies after finishing. This provides a clean way for the user to format log without going into the Pods, especially helpful when the masters are not started or to clean up state without the need to start a master container.

### CODE REVIEW

**Comment by jiacheliu3** [comment]
alluxio-bot, test this please

File: integration/kubernetes/helm/alluxio/values.yaml [hunk 1]
```diff
@@ -105,3 +105,8 @@ const:
 #       medium: ""
 #       size: 1Gi
 #       mountPath: /metastore
+
+# Configuration for Alluxio formatting Jobs
+job:
```
[the remark this answers is not in the document]
Comment (jiacheliu3) [reply integration/kubernetes/helm/alluxio/values.yaml]: SGTM

File: integration/kubernetes/helm/alluxio/templates/alluxio-format-master.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{- $masterCount := int .Values.resources.masterCount }}
```
Comment (jiacheliu3) [inline integration/kubernetes/helm/alluxio/templates/alluxio-format-master.yaml]: @madanadit I have to include some checks to make sure the master can work with the correct config, especially if journaling in HDFS. However the checks become duplicate to what we have in `alluxio-master.yaml` template.

Comment (madanadit) [reply integration/kubernetes/helm/alluxio/templates/alluxio-format-master.yaml]: can we use the helpers file https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm/alluxio/templates/_helpers.tpl?

**Comment by jiacheliu3** [comment]
The document will be included in the [doc PR](https://github.com/Alluxio/alluxio/pull/9976) referring to how the Jobs can be used. Helm template helpers looks promising. I'll try to use that for the duplication. Thanks!

File: integration/kubernetes/helm/alluxio/templates/alluxio-format-master.yaml (hunk 2 above)
Comment (jiacheliu3) [reply integration/kubernetes/helm/alluxio/templates/alluxio-format-master.yaml]: I think the helper file can be applied to generate templates corresponding to sections like Volumes, but the change will be rather big. Can we refactor the duplicate lines in a separate PR?

**Comment by jiacheliu3** [comment]
alluxio-bot, test this please

**Comment by jiacheliu3** [comment]
Sure. [Issue](https://github.com/Alluxio/alluxio/issues/10016) tracked here.

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Pull request merged**